src/controllers/route.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+const home = require('../views/home');
+
+router.get('/', (req, res) => {
+   res.render('home', { activePage: {home: true}});
+});
+
```

Comment:
We've added {{footer}} as our very next step is to build header and footer. Our team has agreed on leaving a indicator for immediate codes to be built. Though we will make sure we don't get ahead of ourselves too much. Thank you Lawson.